domenetjenester/src/main/java/no/nav/foreldrepenger/tilbakekreving/behandling/impl/HalvtRettsgebyrTjeneste.java

```diff
@@ -0,0 +1,56 @@
+package no.nav.foreldrepenger.tilbakekreving.behandling.impl;
+
+import java.time.LocalDateTime;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+
+import no.nav.foreldrepenger.tilbakekreving.behandlingslager.behandling.automatisksaksbehandling.AutomatiskSaksbehandlingRepository;
+import no.nav.foreldrepenger.tilbakekreving.behandlingslager.varsel.VarselRepository;
+import no.nav.foreldrepenger.tilbakekreving.grunnlag.Kravgrunnlag431;
+import no.nav.foreldrepenger.tilbakekreving.grunnlag.KravgrunnlagRepository;
+
+@ApplicationScoped
+public class HalvtRettsgebyrTjeneste {
```

Comment:
Er HåndteringAvSmåFeilutbetalingerTjeneste et bedre navn?

Reply:
Enig. Bruker AutomatiskSaksbehandlingVurderingTjeneste for å matche Batch + Task